subprojects/internal-integ-testing/src/test/groovy/org/gradle/integtests/fixtures/executer/WindowsClassPathMergerTest.groovy

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.executer
+
+import org.gradle.integtests.fixtures.executer.ClassPathMerger.WindowsClassPathMerger
+import org.gradle.util.Requires
+import org.gradle.util.TestPrecondition
+import spock.lang.Specification
+import spock.lang.Subject
+
+import java.util.zip.ZipFile
+
+@Requires(TestPrecondition.WINDOWS)
```

Comment:
Since we are instantiating `WindowsClassPathMerger` directly, do we need to limit the test to only Windows? There is nothing that is specific to Windows.

Reply:
Done.